lib/tests/facts/resolvers/filesystem_resolver.cc

```diff
@@ -14,14 +14,15 @@ using namespace facter::testing;
 
 struct test_filesystem_resolver : filesystem_resolver
 {
-    void add_mountpoint(string name, string device, string filesystem, uint64_t size, uint64_t available, vector<string> options)
+    void add_mountpoint(string name, string device, string filesystem, uint64_t size, uint64_t available, reserved, vector<string> options)
```

Comment:
```suggestion
    void add_mountpoint(string name, string device, string filesystem, uint64_t size, uint64_t available, uint64_t reserved, vector<string> options)
```

This fails because it's missing the variable type